src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -375,8 +377,32 @@ public void parseCommandAliasLocate() throws Exception {
                 LocateCommand.COMMAND_ALIAS + " " + INDEX_FIRST_PERSON.getOneBased());
         assertEquals(new LocateCommand(INDEX_FIRST_PERSON), command);
     }
-
     //@@author
+    //@@author charlesgoh
+    @Test
+    public void parseSortCommand() throws Exception {
```

Comment:
See if we can use heuristic tests to combine the inputs, otherwise we'll need a lot of tests for 1 single sort command.

Reply:
Done 👍